apps/govern/components/NextWeekTooltip.tsx

```diff
@@ -10,14 +11,24 @@ const { Text } = Typography;
 
 const TOOLTIP_STYLE = { maxWidth: '350px' };
 
-export const NextWeekTooltip = ({ children }: { children: ReactNode }) => {
+export const NextWeekTooltip = ({
+  children,
+  hideDescription = false,
+}: {
+  children: ReactNode;
+  hideDescription?: boolean;
+}) => {
   return (
     <Tooltip
       color={COLOR.WHITE}
       overlayStyle={TOOLTIP_STYLE}
       title={
         <Space direction="vertical">
-          <Text>Updated voting weights will take effect at the start of next week Unix time.</Text>
+          {!hideDescription && (
+            <Text>
+              Updated voting weights will take effect at the start of next Unix time week.
```

Comment:
`Updated voting weights will take effect at the start of next week (according to Unix time).`